platform/android/os_android.h

```diff
@@ -70,6 +70,7 @@ class OS_Android : public OS_Unix {
 
 private:
 	Vector<TouchPos> touch;
+	Point2 prev_pos; // needed to calculate the relative position on hover events
```

Comment:
Renaming the variable `hover_prev_pos` will make less ambiguous in the codebase.